.cicd/generate-pipeline.sh

```diff
@@ -342,7 +342,24 @@ EOF
     skip: $(echo "$PLATFORM_JSON" | jq -r '.PLATFORM_SKIP_VAR | env[.] // empty')${SKIP_SERIAL_TESTS}
 
 EOF
-            elif [[ "$TEST_NAME" != 'rodeos_test_eosvmoc' ]]; then
+            elif [[ "$TEST_NAME" != 'rodeos_test_eosvmoc' ]] && \
+                 [[ "$TEST_NAME" != 'rodeos_multi_ship_eos_vm_oc_idle_test' ]] && \
+                 [[ "$TEST_NAME" != 'rodeos_multi_ship_eos_vm_oc_load_test' ]] && \
+                 [[ "$TEST_NAME" != 'rodeos_idle_restart_producer_eos-vm-oc_test' ]] && \
+                 [[ "$TEST_NAME" != 'rodeos_idle_restart_rodeos_eos-vm-oc_test' ]] && \
+                 [[ "$TEST_NAME" != 'rodeos_idle_restart_ship_eos-vm-oc_test' ]] && \
+                 [[ "$TEST_NAME" != 'rodeos_idle_restart_rodeos_producer_eos-vm-oc_test' ]] && \
+                 [[ "$TEST_NAME" != 'rodeos_multi_ship_kill_clean_restart_eos_vm_oc_idle_test' ]] && \
+                 [[ "$TEST_NAME" != 'rodeos_multi_ship_kill_clean_restart_eos_vm_oc_load_test' ]] && \
+                 [[ "$TEST_NAME" != 'rodeos_multi_ship_kill_restart_eos_vm_oc_idle_test' ]] && \
+                 [[ "$TEST_NAME" != 'rodeos_multi_ship_kill_restart_eos_vm_oc_load_test' ]] && \
+                 [[ "$TEST_NAME" != 'rodeos_multi_ship_kill_clean_restart_eos_vm_oc_idle_unixsocket_test' ]] && \
+                 [[ "$TEST_NAME" != 'rodeos_multi_ship_kill_clean_restart_eos_vm_oc_load_unixsocket_test' ]] && \
+                 [[ "$TEST_NAME" != 'rodeos_multi_ship_kill_restart_eos_vm_oc_idle_unixsocket_test' ]] && \
+                 [[ "$TEST_NAME" != 'rodeos_multi_ship_kill_restart_eos_vm_oc_load_unixsocket_test' ]] && \
+                 [[ "$TEST_NAME" != 'rodeos_idle_restart_ship_unix-socket_eos-vm-oc_test' ]] && \
+                 [[ "$TEST_NAME" != 'rodeos_idle_restart_rodeos_unix-socket_eos-vm-oc_test' ]]
```

Comment:
Minor nitpick, but It's cleaner to have multiple expressions inside of of bash test operator, rather than multiple indiepdnet 

Also quotes should be unnecessary 

i.e. 
[[  $TEST_NAME != 'rodeos_test_eosvmoc' &&  \\
     $TEST_NAME != 'rodeos_multi_ship_eos_vm_oc_idle_test'  && \\
     ...
    $TEST_NAME != 'rodeos_idle_restart_rodeos_unix-socket_eos-vm-oc_test'  ]]

Reply:
Thanks. I am making the changes.